plot_app/configured_plots.py

```diff
@@ -573,12 +579,11 @@ def generate_plots(ulog, px4_ulog, db_data, vehicle_data, link_to_3d_page):
                          'Battery Current [A]', 'Discharged Amount [mAh / 100]',
                          'Battery remaining [0=empty, 10=full]'])
     data_plot.change_dataset('system_power')
-    if data_plot.dataset and 'voltage5V_v' in data_plot.dataset.data:
-        if np.amax(data_plot.dataset.data['voltage5V_v']) > 0.0001:
-            data_plot.add_graph(['voltage5V_v'], colors8[7:8], ['5 V'])
-    if data_plot.dataset and 'voltage3V3_v' in data_plot.dataset.data:
-        if np.amax(data_plot.dataset.data['voltage3V3_v']) > 0.0001:
-            data_plot.add_graph(['voltage3V3_v'], colors8[5:6], ['3.3 V'])
+    if data_plot.dataset:
+        if 'voltage5v_v' in data_plot.dataset.data and np.amax(data_plot.dataset.data['voltage5v_v']) > 0.0001:
+            data_plot.add_graph(['voltage5b_v'], colors8[7:8], ['5 V'])
```

Comment:
typo: `voltage5b_v` -> `voltage5v_v`

Can you also check CI? Some lines are too long (you can ignore `C0305: Trailing newlines (trailing-newlines)`)